source/ui/CLS/CLSBeamlineStatusView.cpp

```diff
@@ -72,7 +75,7 @@ void CLSBeamlineStatusView::setBeamlineStatus(CLSBeamlineStatus *newStatus)
 
 void CLSBeamlineStatusView::setSelectedComponent(AMControl *newControl)
 {
-	if (selectedComponent_ != newControl && beamlineStatus_->components().contains(newControl)) {
+	if (selectedComponent_ != newControl && (beamlineStatus_->components().contains(newControl) || beamlineStatus_ == newControl)) {
```

Comment:
When that overall beam status button is clicked, the selected control is the beamlineStatus itself. That's why I added a OR comparison